pom.xml

```diff
@@ -211,13 +162,116 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>migrate-db</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
```

Comment:
This is confusing to me.  So we can't release, build a docker image, run tests, or use the compiler plugin unless we're running the migrate-db profile?